assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -630,10 +630,15 @@ che.workspace.devfile.default_editor.plugins=eclipse/che-machine-exec-plugin/nig
 # Only secrets that match ALL given labels will be selected.
 che.workspace.provision.secret.labels=app.kubernetes.io/part-of=che.eclipse.org,app.kubernetes.io/component=workspace-secret
 
-
 # Plugin is added in case async storage feature will be enabled in workspace config
 # and supported by environment
 che.workspace.devfile.async.storage.plugin=eclipse/che-async-pv-plugin/nightly
 
 # Docker image for the Che async storage
 che.infra.kubernetes.async.storage.image=quay.io/eclipse/che-workspace-data-sync-storage:latest
+
+# Configures name of the trust-store config map where the self-signed certificates are stored.
+che.truststore_configmap_name=ca-certs
```

Comment:
che.truststore.configmap_name?

Reply:
it's because of https://github.com/eclipse/che-operator/blob/master/pkg/deploy/che_configmap.go#L77